src/services/status/status.js

```diff
@@ -13,7 +37,7 @@
 
 var DiyaSelector = require('../../DiyaSelector').DiyaSelector;
 var util = require('util');
-
+const Promise = require('bluebird');
```

Comment:
this dependency should not be bundled into the compiled and minified diya-sdk.js / diya-sdk.min.js files.